manifests/charts/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -201,6 +201,16 @@ spec:
           - name: extracacerts
             mountPath: /cacerts
           {{- end }}
+      affinity:
+      {{- include "nodeaffinity" . | indent 6 }}
+      {{- include "podAntiAffinity" . | indent 6 }}
+{{- if .Values.pilot.tolerations }}
+      tolerations:
+{{ toYaml .Values.pilot.tolerations | indent 6 }}
+{{- else if .Values.global.defaultTolerations }}
```

Comment:
The default value for `global.defaultTolerations` is not defined in `values.yaml`. Actually, the global values are not present in the `2.0` charts so not sure if this condition is applicable here.

Reply:
It's still applicable, the default is defined in the global.yaml file at a higher level

Reply:
Nevermind, found that `defaultTolerations` is already in `global.yaml` file. 
https://github.com/maistra/istio/blob/034f7b7c61ab7027808fccb249857196c0ecf069/manifests/charts/global.yaml#L310

And I think values from `global.yaml` file are now combined into `values.yaml` in`2.1` charts.

Reply:
Upstream PR for combining global values into `values.yaml`. https://github.com/istio/istio/issues/25184